.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
+  - '0.10'
   - '0.12'
-  - 'iojs'
+  - 'node'
```

Comment:
Sounds good!
I need to go. I'll force-push to this branch later tonight or tomorrow.